satellite/console/service.go

```diff
@@ -187,14 +187,30 @@ func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken st
 		return nil, err
 	}
 
+	var customerID []byte
 	userPayments, err := s.store.UserPayments().Get(ctx, authorization.User.ID)
 	if err != nil {
```

Comment:
If/else?

Reply:
Add payment method shouldn't create `project to payment method mapping`

Reply:
Renamed to AddNewProjectPaymentMehtod